src/js/new-visualization-modal.js

```diff
@@ -54,7 +47,54 @@ export default function initModal() {
   // add SQL query to the modal
   document.getElementById('data-query').innerHTML = dataQuery;
 
+  hideAllWarnings();
+
   // wire up event listeners
   $('#new-vis-button').click(onNewVisSubmit);
   $('#copy-query-button').click(onCopyQueryClick);
 }
+
+const csvStringToArray = (strData) => {
+  const objPattern = new RegExp('(\\,|\\r?\\n|\\r|^)(?:"([^"]*(?:""[^"]*)*)"|([^\\,\\r\\n]*))', 'gi');
+  // Create an array to hold our individual pattern
+  // matching groups.
+  let arrMatches = null,
+    // Create an array to hold our data. Give the array
```

Comment:
no reason to have such narrow comments. Collapse into single line comment please

Reply:
do this for all similar comment breaks in this section

Reply:
Done.